src/main/resources/userEmailTemplate-feedbackSession.html

```diff
@@ -4,9 +4,58 @@
 
 <p>
     The following feedback session ${status}.
-    <br>&nbsp;&nbsp; Course: [${courseId}]${courseName} 
-    <br>&nbsp;&nbsp; Feedback Session Name: ${feedbackSessionName}
-    <br>&nbsp;&nbsp; Deadline: ${deadline}
+    <div>
+        <table style="max-width:600px;border:1px solid black;border-collapse:collapse;margin-left:10px;">
+            <tr>
+                <th colspan="2" style="padding:10px;text-align:center;font-size:120%;">
+                    Feedback Session Details
+                </th>
+            </tr>
+            <tr>
+                <td style="padding:10px;border:1px solid black;">
+                    <strong>
+                        Course:
+                    </strong>
+                </td>
+                <td style="padding:10px;border:1px solid black;">
+                    [${courseId}] ${courseName} 
+                </td>
+            </tr>
+
+            <tr>
+                <td style="padding:10px;border:1px solid black;">
+                    <strong>
+                        Feedback Session Name:
+                    </strong>
+                </td>
+                <td style="padding:10px;border:1px solid black;">
+                    ${feedbackSessionName}
+                </td>
+            </tr>
+
+            <tr>
+                <td style="padding:10px;border:1px solid black;">
+                    <strong>
+                        Deadline:
+                    </strong>
+                </td>
+                <td style="padding:10px;border:1px solid black;">
+                    ${deadline}
+                </td>
+            </tr>
+
+            <tr>
+                <td style="padding:10px;border:1px solid black;">
+                    <strong>
+                        Session Instructions:
+                    </strong>
+                </td>
+                <td style="padding:10px;border:1px solid black;">
+                    ${sessionInstructions}
+                </td>
+            </tr>
+        </table>
```

Comment:
Here's how it looks without padding and without border:
<img width="1280" alt="screen shot 2017-04-16 at 5 51 39 pm" src="https://cloud.githubusercontent.com/assets/18088721/25071139/83cb32c8-22cd-11e7-95ca-ab98f8e21889.png">

Here's how it looks with padding and without border:
<img width="1280" alt="screen shot 2017-04-16 at 5 52 10 pm" src="https://cloud.githubusercontent.com/assets/18088721/25071154/c2983532-22cd-11e7-9a2d-aef41bd15583.png">

In my opinion, the inline styles on the `<table>` tag (i.e. `max-width`, `border`, `border-collapse` and `margin-left`), the `padding` on all the table elements and the `text-align:center` on the table header are essential. We could get rid of the borders on the rest of the `<td>'s` if you so wish and the `font-size` is just to distinguis the table header from the row headers.